shared/concatenateMuseMarkers.m

```diff
@@ -0,0 +1,49 @@
+function [marker_clock, marker_synctime] = concatenateMuseMarkers(MuseStruct, ipart, markerName)
```

Comment:
For now it seems better to keep this in your own project/functions directory, or under /develop, as this introduces another way of organizing the marker data which is inconsistent with the rest of the functions, and not used like this yet. Also inconsisent use of the input arguments with respect to the existing code.